### ISSUE TITLE
wtf?

### ISSUE THREAD

**Issue by ryantheleach** [bootstrap-tagsinput/bootstrap-tagsinput#14]
I've got no idea what happened, but my github inbox was flooded with mentions/updates from this repo, that led to 404's.

An issue or discussion or something in the readme may help others in the same situation.

I assume that something migrated, somehow in a way that github doesn't have elegant support for.

**Comment by filipkappa** [comment]
Yes, we have some missing releases - they will be recovered next week,

**Comment by juneidysoo** [comment]
I get the frustration. I totally agree that someone messed up the repo by rewriting the history.

But honestly, the aggression from the title is uncalled for. This is open source repo and the owner/contributor provided access to repo without any warranty.

**Comment by ryantheleach** [comment]
There's no hostility intended. 

I was looking for an explanation was all, and still haven't got one, in a location where others would find it, given that I have zero knowledge of what has occured, I had no idea what to title it. 

Apologies if you are used to a more formal environment, but I was merely being colloquial. 

All I've seen is an acknowledgement of missing releases.

From an outside perspective, from someone who hasn't played with this repo in 4 years, it looks suspiciously like a supply chain attack given the erasure of history etc.

I think an explanation is warranted.

**Comment by filipkappa** [comment]
The mistake is 100% on our side, and I understand your frustration, especially if some of the projects relied on the missing versions. As I've said, all of them will be restored soon - this time we'll do it with care to avoid such situations.

I changed your title only to make it more descriptive and easily findable for people facing similar issues.

**Comment by jcrben** [comment]
Will the deleted issues be restored?

**Comment by ryantheleach** [comment]
Once again, I'm not just talking about the releases.

For what it's worth, this is what my github inbox currently looks like, just in case there's a misunderstanding.

![image](https://user-images.githubusercontent.com/141554/162723560-5834c365-75f1-4666-9c21-d6c91c34998c.png)

The commit history currently looks like:

![image](https://user-images.githubusercontent.com/141554/162723695-cf2cc5c5-df99-4e54-b900-7ecda96e10e4.png)

as opposed to a branch (that may be out of date, who knows) showing the more helpful previous history.

https://github.com/bootstrap-tagsinput/bootstrap-tagsinput/commits/develop

So there's a loss of history in the main branch.

You have acknowledged the issue, but still are not admitting to what caused it.
Whether there's been a change of maintainers, etc.
Whether you are trying to restore a backup, or if this is the new normal.
Or whether you are waiting on Github Support.

People are willing to be patient, but what they currently are, is confused.

**Comment by ryantheleach** [comment]
At this point seeing you continue to "migrate" the repo, without explanation, I'm forced to conclude that you have swapped the names of 2 repo's, in order to farm 'stars' for the bootstrap-templates project, or something equally shady.
